frontend/src/Globalstyles.jsx

```diff
@@ -1,23 +1,35 @@
 import { createGlobalStyle } from 'styled-components';
 
-
 const GlobalStyle = createGlobalStyle`
   :root {
-    --primary-color: #fff;
-    --secondary-color: #6c757d;
-    --success-color: #28a745;
-    --danger-color: #dc3545;
-    --warning-color: #ffc107;
-    --info-color: #17a2b8;
-    --light-color: #f8f9fa;
-    --dark-color: #343a40;
+    --ego-white: #fff;
+    --ego-green: #687943;
+    --ego-lgt-green: #CCE0A1;
+    --ego-beige: #DCDED0;
+    --ego-purple: #C590FB;
+    --ego-error: #794343;
```

Comment:
Yeah to global styles. 
Did you add new colors? This one is not in the design.

Reply:
Great to the global variables!
Yeah, I also noticed. the error box color is also --ego-purple: #C590FB; :)

Reply:
added some who I thought we needed . will fix the error color :)